hedgehog/src/Hedgehog/Internal/Config.hs

```diff
@@ -166,6 +172,12 @@ detectWorkers = do
       Just env ->
         pure $ WorkerCount env
 
+detectSkip :: MonadIO m => m Skip
+detectSkip =
+  liftIO $ do
+    menv <- (skipDecompress =<<) <$> lookupEnv "HEDGEHOG_SKIP"
+    pure $ fromMaybe SkipNothing menv
```

Comment:
Invalid values here are silently ignored. That's consistent with the others, but I kind of feel like an error would be friendlier?

Reply:
yeah all of the env vars are a bit sketchy like that! so yeah add an error message to this one 👍 

This whole module probably needs a bit of an overhaul so we just read and parse all the env vars once and report warnings/errors a single time.

I'm happy for you to just `error` out though for this PR.